extension/content/less/cm-ndt.less

```diff
@@ -44,6 +44,8 @@
     border: 1px solid var(--cm-border);
     border-radius: @input-border-radius;
     color: var(--cm-color);
+    font-family: @font-family-monospace;
+    font-variant-ligatures: no-contextual;
```

Comment:
Not a fan of ligatures?

Reply:
I am but I think they should be optional with "off" being the default.

Reply:
I'll be adding UI for extension settings that will let us customize some of the appearance stuff like this.